src/jsonSchema.ts

```diff
@@ -79,7 +79,8 @@ export const jsonSchemaResolver = <TFieldValues extends Record<string, any>>(
     // compile only if necessary. otherwise return the memoized validation function
     if (!validate || allErrors !== validateAllFieldCriteria) {
       allErrors = validateAllFieldCriteria;
-      validate = new Ajv({
+      // @ts-ignore
+      validate = new Ajv.default({
```

Comment:
This shouldn't be done. TS was made by very clever people who know how the modules systems work. If it's an error, it won't work on runtime. In this case changing the import and changing this line back to how it was will remove the error.

Reply:
i build it and failed without it.